<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoControlWidget</class>
 <widget class="QWidget" name="VideoControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Frame</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="_frameSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="_downButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="_frameSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="_upButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>193</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_frameSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>VideoControlWidget</receiver>
   <slot>spin_box_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_downButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoControlWidget</receiver>
   <slot>down_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_frameSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>VideoControlWidget</receiver>
   <slot>slider_moved()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_upButton</sender>
   <signal>clicked()</signal>
   <receiver>VideoControlWidget</receiver>
   <slot>up_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
